org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/ExprUnary.java

```diff
@@ -38,6 +38,8 @@
  * Immutable; represents a unary expression of the form "(OP subexpression)"
  * <p>
  * <b>Invariant:</b> type!=EMPTY => sub.mult==0
+ *
+ * @modified Eduardo Pessoa, Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
adds unary temporal ops for formulas (after, always, eventually, before, historically, once) and expressions (primes)